server.cfg

```diff
@@ -35,6 +35,8 @@ set qbx:max_gangs_per_player 1 # Sets the number of gangs per player
 set qbx:setjob_replaces "true" # When true, the SetJob function deletes the previous primary job of the player before setting the new one
 set qbx:setgang_replaces "true" # When true, the SetGang function deletes the previous primary gang of the player before setting the new one
 set qbx:cleanplayergroups "true" # When true, on server startup, the database will be checked for any groups/grades players have that are not in config files and removes them from the database if so.
+setr qbx:motdTemplate "<p style='padding: 1rem; background-color: rgb(30, 30, 30); border-radius: 1rem; text-align: center;'><span style='color: rgb(40, 200, 40);'>{0}</span><br>{1} <span id='link' style='color: rgb(100, 100, 255);' onclick='window.invokeNative(\"openUrl\", \"{2}\")'>{2}</span></p><style>span#link:hover { cursor: pointer; }</style>"
+setr qbx:motdArgs ["Welcome to Qbox!", "Learn more here:", "https://docs.qbox.re/"]
```

Comment:
Is there a way we can make the motd a single string argument? This feels a bit too in the weeds, I assume people could write the style inline if it were a single string?